internal/pkg/service/common/etcdop/op/op_test.go

```diff
@@ -21,7 +21,7 @@ type testOp struct {
 }
 
 type testValue struct {
-	Foo string
+	Foo string `json:"foo"`
 }
 
```

Comment:
:man_shrugging: apparently it is not necessary here

Reply:
Should we ommit the `test` files from this linter?